<launch>
	<node name="ar_pose" pkg="ar_pose" type="ar_single" respawn="false" output="screen">
		<remap from="/usb_cam/image_raw" to="/camera/rgb/image_color" />
		<remap from="/usb_cam/camera_info" to="/camera/rgb/camera_info" />
		
		<param name="marker_pattern" type="string" value="$(find pole_structure_mapper)/data/patt.lowergripper"/>
		<param name="marker_width" type="double" value="175.0"/>
		<param name="marker_center_x" type="double" value="0.0"/>
		<param name="marker_center_y" type="double" value="0.0"/>
		<param name="threshold" type="int" value="100"/>
		<param name="use_history" type="bool" value="true"/>
		<param name="marker_frame" type="string" value="ar_marker"/>
		<param name="reverse_transform" type="bool" value="false"/>
	</node>
</launch>
